extensions/binding/org.eclipse.smarthome.binding.triggertest/ESH-INF/thing/thing-types.xml

```diff
@@ -51,6 +51,7 @@
         <kind>trigger</kind>
         <label>Trigger 2</label>
         <description>Description of trigger 2</description>
+        <event/>
```

Comment:
The framework doesn't validate the sent events. Regardless of what you declare in the XML, you can send any event. If you call `triggerChannel(ChannelUID)`, an empty event `""` is sent.